alternatives.c

```diff
@@ -485,7 +485,7 @@ static int readConfig(struct alternativeSet *set, const char *title,
             newAlt.followers[i - 1].target = (line && strlen(line)) ? strsteal(&line) : NULL;
         }
 
-        if (newAlt.priority > set->alts[set->best].priority)
+        if (set->best == -1 || newAlt.priority > set->alts[set->best].priority)
```

Comment:
This could also be moved a bit down beyond line 493. Then it would continue to work with the initial `0`.

Reply:
Yep, you are right, feel free to force-push a removal of this one and amend your old one.

Reply:
Done.